samples/bot-sequential-flow-adaptive-cards/csharp/SequentialUserSpecificFlow/Bots/UserSpecificBot.cs

```diff
@@ -161,15 +258,86 @@ protected override async Task<InvokeResponse> OnInvokeActivityAsync(ITurnContext
                         adaptiveCardResponse = new AdaptiveCardInvokeResponse()
                         {
                             StatusCode = 200,
-                            Type = "application/vnd.microsoft.card.adaptive",
+                            Type = AdaptiveCard.ContentType,
                             Value = response
                         };
+                        
                         return CreateInvokeResponse(adaptiveCardResponse);
                 }
             }
+
             return null;
         }
 
+        private Attachment GetFirstOptionsAdaptiveCard(string[] filepath, string name = null, string userMRI = null)
+        {
+            var adaptiveCardJson = File.ReadAllText(Path.Combine(filepath));
+            AdaptiveCardTemplate template = new AdaptiveCardTemplate(adaptiveCardJson);
+            var payloadData = new
+            {
+                createdById = userMRI,
+                createdBy = name
+            };
+            var cardJsonstring = template.Expand(payloadData);
+            var adaptiveCardAttachment = new Attachment()
+            {
+                ContentType = AdaptiveCard.ContentType,
+                Content = JsonConvert.DeserializeObject(cardJsonstring),
+            };
+
+            return adaptiveCardAttachment;
+        }
```

Comment:
Could you please move all the card creation code into CardHelper class? Move all the functions which are returning card/attachments.

Reply:
Moved to helper class.